### PR TITLE
Doc: Added setting up and signup flow in readme

### PR DESCRIPTION
**Pull request by shubhamji88**
## 🚧 Type of change

- Document update

## 📔 Objective

Update Readme.md

## 📋 Code changes

- **Readme.md:** Added setting up this project and how signup flow is working.

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

**Comment by abergs** [comment]
Hey @shubhamji88, thank you for the contributions. We recently activated what's called a "CLAbot" on this repository, it's a bot that automates a document (Contributor License Agreement) where you agree that you're allowed to contribution code to this repository and is something we must do for all our repositories at Bitwarden.

It should automatically show up as a comment in your PR, but since it haven't yet... could you try to make another commit to this PR to see if it prompts the automation? Or, if that doesn't work: Close this PR and open a new one?

Thanks for your contributions to this.

File: README.md [hunk 1]
```diff
@@ -11,11 +11,47 @@ Passwordless.dev consists of three key parts:
 
 ## Requirements
 * Android 9+ (API 28)
+## Setting up Passwordless-android
+1. Generate SHA256 for your app. Refer [link](https://stackoverflow.com/questions/42290681/android-studio-only-gives-me-sha1-i-need-sha256)
```
Comment (sujan6539) [inline README.md:15]: As opposed to referencing a  stackoverflow link in md file, isn't it better to paste the snippet into the file?

Comment (sujan6539) [reply README.md:15]: I see! You've pasted the solution in line 20.

Comment (shubhamji88) [reply README.md:15]: Sure, that's actually two different steps. will make a new PR due to CLAbot activation. And fix this

**Pull request closed without merging**